h_transport_materials/property_database/niobium.py

```diff
@@ -25,6 +24,39 @@
     isotope="T",
 )
 
+qi_diffusivity_d = Diffusivity(
+    D_0=5.2e-4 * u.cm**2 * u.s**-1,
+    E_D=0.127 * u.eV * u.particle**-1,
+    range=(
+        u.Quantity(-140, u.degC),
+        u.Quantity(100, u.degC),
+    ),
+    source="qi_tritium_1983",
+    isotope="D",
+)
+
+qi_diffusivity_h_high_temp = Diffusivity(
+    D_0=5e-4 * u.cm**2 * u.s**-1,
+    E_D=0.106 * u.eV * u.particle**-1,
+    range=(
+        250 * u.K,
+        u.Quantity(100, u.degC),
+    ),
+    source="qi_tritium_1983",
+    isotope="H",
+)
+
+qi_diffusivity_h_low_temp = Diffusivity(
+    D_0=0.9 * u.cm**2 * u.s**-1,
+    E_D=0.068 * u.eV * u.particle**-1,
+    range=(
+        u.Quantity(-160, u.degC),
```

Comment:
qi_diffusivity_h_low_temp should have a range from -140 C to 250 K.

Reply:
Should we trust the graph rather than the text? all materials and isotopes have different ranges